### ISSUE TITLE
when multi entry includes 'index',webpack entry config does not work

### ISSUE THREAD

**Issue by pdd-jiama** [webpack/webpack#10498]
# Bug report

**What is the current behavior?**
webpack.config.config.js
```js
const {CleanWebpackPlugin} = require('clean-webpack-plugin');
module.exports = {
    mode: 'development',
    entry: {
        home: './home.js',
        index: './index.js'
    },
    output: {
        filename: "[name].bundle.js"
    },
    plugins: [
        new CleanWebpackPlugin()
    ]
}
```
I cann't get 2 bundle as I expected.if I rename 'index' to 'home2'，it works.
**If the current behavior is a bug, please provide the steps to reproduce.**
##### my directory Structure is below:
├── home.js
├── index.js
├── package-lock.json
├── package.json
├── readme.md
└── webpack.config.js
##### files content:
 index.js
```js
console.log('index.js');
```
home.js
```js
console.log('home.js');
```
just run 'webpack' in the root directory

**What is the expected behavior?**
get 2 bundle output, as below:
'home.bundle.js'
'index.bundle.js',
but I just get below one; 
'main.bundle.js',

**Other relevant information:**
webpack version:  4.42.0("webpack-cli": "^4.0.0-beta.2")
Node.js version: v12.13.0
Operating System: macOs
Additional tools:

**Comment by sokra** [comment]
Thanks for your report. It would be great if you reduce your issue to a small reproducible example. Best put this example into a github repository together with instructions how to get to the problem.

**Comment by pdd-jiama** [comment]
> Thanks for your report. It would be great if you reduce your issue to a small reproducible example. Best put this example into a github repository together with instructions how to get to the problem.

it is easy to plicate the problem, if you have a index.js in you root diretory, the 'entry' will be configed as index.js, and get a result 'main.bundle.js', but my entry config is below:
```js
    entry: {
        home: './home.js',
        index: './index.js'
    },
```
i expect to get 
```js
     home.bundle.js
    index.bundle.js
```
you can just try with a lite demo

**Comment by Mistyyyy** [comment]
you should use webpack-cli with the latest version (3.3.11) rather than 4.0.0-beta.2.
this version has been deprecated due to bad publishing so you can't find this version in [npm.js](https://www.npmjs.com/package/webpack-cli)

**Comment by ysj16** [comment]
It perform normally in my environment, maybe you should provide a complete demo.

**Comment by Mistyyyy** [comment]
The issue is fixed in webpack/webpack-cli#1289, @evilebottnawi you can close this issue.
